### DISCUSSION TITLE
Account Suspended

### DISCUSSION THREAD

**Discussion by Shikimoriix** [community/community#83994] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Dear GitHub Support Team,

I hope this message finds you well. It has been over a month, and my GitHub account, [@ikx7a](www.github.com/ikx7a), is still suspended. Unfortunately, I have not received any response regarding the reason for the suspension.

This suspension has led to the deletion of all my projects, causing significant disruption. I kindly request immediate assistance in resolving this matter and understanding the grounds for the account suspension. Your prompt response and support in recovering my projects would be greatly appreciated.

For your reference, my contact email is ikx7a@hotmail.com. I am eager to continue contributing to the GitHub community, and your assistance in this matter is crucial.

Thank you for your attention to this urgent issue.

Sincerely,
[ikx7a](www.github.com/ikx7a)

**Comment by quangzn** [comment]
Dear Github team,
My github id suddenly suspended,
Please solved the problem as soon as possible. I purchased the subscription for github copilot. 
My ticket id-2567406

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Shikimoriix, 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
